deploy/helm/sumologic/conf/logs/collector/otelcol/config.yaml

```
@@ -1 +1,245 @@
+      ## Input Attributes:
+      ## - log.file.path: '/var/log/pods/default_logger-multiline-4nvg4_aed49747-b541-4a07-8663-f7e1febc47d5/loggercontainer/0.log'
+      ## Output Attributes:
+      ## - log.file.path: '/var/log/pods/default_logger-multiline-4nvg4_aed49747-b541-4a07-8663-f7e1febc47d5/loggercontainer/0.log'
+      ## - container_name: "loggercontainer",
+      ## - namespace: "default",
+      ## - pod_name: "logger-multiline-4nvg4",
+      ## - run_id: "0",
+      ## - uid: "aed49747-b541-4a07-8663-f7e1febc47d5"
+      ## }
+      - id: extract-metadata-from-filepath
+        type: regex_parser
+        regex: '^.*\/(?P<namespace>[^_]+)_(?P<pod_name>[^_]+)_(?P<uid>[a-f0-9\-]+)\/(?P<container_name>[^\._]+)\/(?P<run_id>\d+)\.log$'
+        parse_from: attributes["log.file.path"]
+
+      ## The following actions are being performed:
+      ## - renaming attributes
+      ## - moving stream from body to attribtues
+      ## - using body.log as body
+      ## - create fluent.tag attribute in order to route in metadata pods
+      ## Input Body (JSON): {
+      ##   "log": "2001-02-03 04:05:06 loggerlog 1 first line\n",
+      ##   "stream": "stdout",
+      ## }
+      ## Output Body (String): "2001-02-03 04:05:06 loggerlog 1 first line\n"
+      ## Input Attributes:
+      ## - log.file.path: '/var/log/pods/default_logger-multiline-4nvg4_aed49747-b541-4a07-8663-f7e1febc47d5/loggercontainer/0.log'
+      ## - container_name: "loggercontainer",
+      ## - namespace: "default",
+      ## - pod_name: "logger-multiline-4nvg4",
+      ## - run_id: "0",
+      ## - uid: "aed49747-b541-4a07-8663-f7e1febc47d5"
+      ## Output Attributes:
+      ## - k8s.container.name: "loggercontainer"
+      ## - k8s.namespace.name: "default"
+      ## - k8s.pod.name: "logger-multiline-4nvg4"
+      ## - k8s.pod.uid: "aed49747-b541-4a07-8663-f7e1febc47d5"
+      ## - run_id: "0"
+      ## - stream: "stdout"
+      ## - fluent.tag: "containers.loggercontainer"
+      - id: move-attributes
+        type: move
+        from: body.stream
+        to: attributes["stream"]
+      - type: move
+        from: attributes.container_name
+        to: attributes["k8s.container.name"]
+      - type: move
+        from: attributes.namespace
+        to: attributes["k8s.namespace.name"]
+      - type: move
+        from: attributes.pod_name
+        to: attributes["k8s.pod.name"]
+      - type: move
+        from: attributes.run_id
+        to: attributes["run_id"]
+      - type: move
+        from: attributes.uid
+        to: attributes["k8s.pod.uid"]
+      - type: add
+        field: attributes["fluent.tag"]
+        value: EXPR("containers." + attributes["k8s.container.name"])
+      ## Use remove operator when available in opentelemetry collector:
+      ## https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9524
+      - type: move
+        from: attributes["log.file.path"]
+        to: body["log.file.path"]
```

remove operator is already supported

```suggestion
      - type: remove
        field: attributes["log.file.path"]
```

---

Done.